distribution/openrct2.d.ts

```diff
@@ -2606,6 +2606,56 @@ declare global {
         "waiting_to_depart" |
         "waiting_to_start";
 
+    type CrashParticleType =  0 | 1 | 2 | 3 | 4;
+
+    /**
+     * Represents a vehicle explosion particle. They are emitted during a vehicle
+     * crash and will bounce until their timer expires and they are automatically
+     * removed.
+     */
+    interface CrashedVehicleParticle extends Entity {
+        /**
+         * The colour of the particle. Only body and trim are used.
+         */
+        colours: VehicleColour;
+
+        /**
+         * The lifetime of the particle in ticks. Default value 65535. Entity is
+         * automatically removed at 0.
+         */
+        timeToLive: number;
+
+        /**
+         * The particle velocity.
+         */
+        velocity: CoordsXYZ;
+
+        /**
+         * The acceleration of the particle in the x, y, and z directions.
+         */
+        acceleration: CoordsXYZ;
+
+        /**
+         * The type of crash particle.
+         */
+        crashParticleType: CrashParticleType;
+
+        /**
+         * The current frame of the crash particle.
+         */
+        frame: number;
```

Comment:
What is this one exactly? I expected it to increment by 1 every game tick but it actually increments by 85 every game tick, until about ~3100, after which it wraps back to 0.

Reply:
It's a weird one. There's 12 animation sprites per particle type, and the frame number is divided by 256 to get the sprite number. At incrementing 85 per tick it animates at 3 ticks per sprite.

Reply:
It sounds like a timer and a frame number in one field. 😄 Is it worth it to do that same divide here, so it's really just the frame number?

Reply:
This question doesn’t appear to be answered yet?

Reply:
Basssiiieee's suggestion was implemented as a multiplier/divisor of kCrashedVehicleParticleFrameToSprite in the getter/setters for the property.

Reply:
For me it's answered with the implementation of the multiplication/division yes. :)